### PR TITLE
Practicing GitHub flow

### PR DESCRIPTION
**Pull request by greiser**
This pull request attempts to demonstrate GitHub flow using CLI and GitHub

### CODE REVIEW

File: gwr-bio.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+### Hometown
+Washington D.C.
```
Comment (kodali9) [inline gwr-bio.md]: testing line level comment

Comment (sree2016) [inline gwr-bio.md]: Hows the weather in DC?

File: gwr-bio.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+### Hometown
```
Comment (andre-l-mm) [inline gwr-bio.md]: test line level comment

**Comment by kodali9** [comment]
@greiser  We are from the same town

File: gwr-bio.md (hunk 1 above)
Comment (eitah) [inline gwr-bio.md]: this is a cool city! I'm from Chicago.

**Comment by lranasinghe** [comment]
@greiser  TEST

**Comment by sree2016** [comment]
@greiser this is a comment

File: gwr-bio.md (hunk 1 above)
Comment (greiser) [inline gwr-bio.md]: @sree2016 Rainy today
@eitah Thanks, I love Chicago too

**Comment by greiser** [comment]
@kodali9 :sunglasses: :+1:

**Pull request merged**